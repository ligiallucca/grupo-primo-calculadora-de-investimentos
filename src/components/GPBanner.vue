<template>
  <div class="banner">
    <div class="backdrop">
      <h1>Simulador de investimento</h1>
      <p>Descubra o quanto vocÃª pode economizar.</p>
    </div>
  </div>
</template>
<style lang="scss" scoped>
@import '../assets/main.scss';
.banner {
  // height: 347px;
  color: $white;
  background-image: url('../assets/images/header-bg.jpg');
  background-size: cover;
  background-color: $dark;


  .backdrop {
    background-color: $dark;
    height: 100%;
    padding: 80px 120px;
    opacity: 0.9;
  }
  h1 {
    font-size: 48px;
    font-weight: 800;
    line-height: 58px;
    margin-bottom: 8px;
    @include for-small-devices {
      font-size: 26px;
      line-height: 31px;
      margin-bottom: 16px;
    }
  }
  p {
    font-size: 24px;
    font-weight: 400;
    line-height: 29px;
    @include for-small-devices {
      font-size: 14px;
      line-height: 16px;
    }
  }
}
</style>
