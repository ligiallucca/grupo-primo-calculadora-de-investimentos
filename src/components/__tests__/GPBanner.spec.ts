import { describe, it, expect } from 'vitest'

import { mount } from '@vue/test-utils'
import GPBanner from '../GPBanner.vue'

describe('GPBanner', () => {
    it('renders properly', () => {
        const wrapper = mount(GPBanner)
        expect(wrapper.html).toMatchSnapshot()
        expect(wrapper.text()).toContain('investimento')
    })
})