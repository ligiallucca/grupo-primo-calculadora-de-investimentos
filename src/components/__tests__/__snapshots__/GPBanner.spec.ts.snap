// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`GPBanner > renders properly 1`] = `[Function]`;
